intuita-webview/src/jobDiffView/DiffViewer/Header/index.tsx

```diff
@@ -33,10 +33,10 @@ export const Header = (props: Props) => {
 							/>
 						</VSCodeButton>
 					}
-					popoverText="Move to the previous file"
+					content="Move to the previous file"
 				/>
-				<Popover
-					trigger={
+				<IntuitaPopover
```

Comment:
I think it can be more readable if we wrap children with IntuitaPopover like this: 
```
<IntuitaPopover> 
  <VsCodeButton />
  </IntuitaPopover>
  ```
  instead of passing `children` prop. 1-st way feels more natural.